articles/azure-vmware/configure-storage-policy.md

```diff
@@ -73,7 +73,8 @@ You'll run the `Get-StoragePolicy` cmdlet to list the vSAN based storage policie
 
 ## Set storage policy on VM
 
-You'll run the `Set-VMStoragePolicy` cmdlet to Modify vSAN based storage policies on an individual VM or on a group of VMs sharing a similar VM name. For example, if you have 3 VMs named "MyVM1", "MyVM2", "MyVM3", supplying "MyVM*" to the VMName parameter would change the StoragePolicy on all three VMs.
+You'll run the `Set-VMStoragePolicy` cmdlet to Modify vSAN based storage policies on an default cluster or individual VM or on a group of VMs sharing a similar VM name. For example, if you have 3 VMs named "MyVM1", "MyVM2", "MyVM3", supplying "MyVM*" to the VMName parameter would change the StoragePolicy on all three VMs.
```

Comment:
@prasad3017 please change 'an' to 'a' in the first sentence and use oxford commas in same sentence taking out the first 'or' between them.
ex: You'll run the `Set-VMStoragePolicy` cmdlet to Modify vSAN based storage policies on a default cluster, individual VM, or on a group of VMs sharing a similar VM name.